src/Examples/UnitModels/Reactors/equilibrium_reactor_testing.ipynb

```diff
@@ -0,0 +1,1138 @@
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 4,
+   "metadata": {
+    "scrolled": true
+   },
+   "outputs": [],
+   "source": [
+    "thermo_props_config_dict = get_prop(components=[\"CH4\", \"H2O\", \"H2\", \"CO\", \"CO2\"])\n",
+    "m.fs.thermo_params = GenericParameterBlock(default=thermo_props_config_dict)\n",
+    "m.fs.reaction_params = GenericReactionParameterBlock(default={\"property_package\": m.fs.thermo_params,\n",
+    "                                                              **reaction_props.config_dict})"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Adding Unit Models\n",
+    "\n",
+    "Let us start adding the unit models we have imported to the flowsheet. Here, we are adding a Mixer (assigned a name M101), a compressor (assigned a name C101), a Heater (assigned a name H101) and an EquilibriumReactor (assigned a name R101). Note that all unit models should be explicitly defined with a given property package. In addition to that, there are several arguments depending on the unit model, please refer to the documentation for more details on [IDAES Unit Models](https://idaes-pse.readthedocs.io/en/stable/reference_guides/model_libraries/index.html). For example, the Mixer unit model here is given a `list` consisting of names to the two inlets. Note that the Compressor unit model is a PressureChanger unit model assuming compression operation and with a fixed isentropic compressor efficiency as the default thermodynamic behavior."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 5,
+   "metadata": {
+    "scrolled": true
+   },
+   "outputs": [],
+   "source": [
+    "m.fs.M101 = Mixer(default={\"property_package\": m.fs.thermo_params,\n",
+    "                           \"inlet_list\": [\"methane_feed\", \"steam_feed\"]})\n",
+    "m.fs.H101 = Heater(default={\"property_package\": m.fs.thermo_params,\n",
+    "                            \"has_pressure_change\": False,\n",
+    "                            \"has_phase_equilibrium\": False})\n",
+    "m.fs.C101 = Compressor(default={\"property_package\": m.fs.thermo_params})"
+   ]
```

Comment:
Since this is an official Example, we'll want to originate the feed streams in `Feed` blocks and link the product streams to a `Product` block.